[2017-10-12 08:54:17,897][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 08:54:17,906][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 10:21:02,445][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 10:21:02,455][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 10:22:10,015][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 10:22:10,025][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 10:22:46,393][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 10:22:46,402][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 10:24:58,688][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 10:24:58,696][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 10:27:12,720][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 10:27:12,730][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 10:46:10,962][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 10:46:10,970][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 10:53:55,366][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 10:53:55,374][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 10:54:48,300][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 10:54:48,309][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 10:54:57,633][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 10:54:57,643][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 10:55:07,598][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 10:55:07,606][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:02:39,435][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:02:39,443][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:03:51,241][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:03:51,249][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:05:49,484][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:05:49,494][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:06:39,858][TxnService-3]com.ods.db.DbPool[60][ERROR]: 获取数据库连接失败
java.sql.SQLRecoverableException: IO 错误: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:489)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:553)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:254)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:32)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:528)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:149)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:217)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:143)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1466)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1529)
	at com.alibaba.druid.pool.DruidDataSource.init(DruidDataSource.java:822)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1172)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1168)
	at com.ods.db.DbPool.getConnection(DbPool.java:57)
	at com.ods.db.DbQuery.excuteQuery(DbQuery.java:36)
	at com.ods.transaction.test.transaction(test.java:24)
	at com.ods.service.TxnService.run(TxnService.java:175)
Caused by: oracle.net.ns.NetException: The Network Adapter could not establish the connection
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:439)
	at oracle.net.resolver.AddrResolution.resolveAndExecute(AddrResolution.java:454)
	at oracle.net.ns.NSProtocol.establishConnection(NSProtocol.java:693)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:251)
	at oracle.jdbc.driver.T4CConnection.connect(T4CConnection.java:1140)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:340)
	... 16 more
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at oracle.net.nt.TcpNTAdapter.connect(TcpNTAdapter.java:149)
	at oracle.net.nt.ConnOption.connect(ConnOption.java:133)
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:405)
	... 21 more
[2017-10-12 11:06:39,861][TxnService-3]com.ods.db.DbPool[62][DEBUG]: 数据库连接获取完毕
[2017-10-12 11:08:57,602][TxnService-4]com.ods.db.DbPool[60][ERROR]: 获取数据库连接失败
com.alibaba.druid.pool.GetConnectionTimeoutException: wait millis 60000, active 0, maxActive 10, creating 0
	at com.alibaba.druid.pool.DruidDataSource.getConnectionInternal(DruidDataSource.java:1377)
	at com.alibaba.druid.pool.DruidDataSource.getConnectionDirect(DruidDataSource.java:1198)
	at com.alibaba.druid.filter.FilterChainImpl.dataSource_connect(FilterChainImpl.java:4619)
	at com.alibaba.druid.filter.stat.StatFilter.dataSource_getConnection(StatFilter.java:666)
	at com.alibaba.druid.filter.FilterChainImpl.dataSource_connect(FilterChainImpl.java:4615)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1176)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1168)
	at com.ods.db.DbPool.getConnection(DbPool.java:57)
	at com.ods.db.DbQuery.excuteQuery(DbQuery.java:36)
	at com.ods.transaction.test.transaction(test.java:24)
	at com.ods.service.TxnService.run(TxnService.java:175)
Caused by: java.sql.SQLRecoverableException: IO 错误: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:489)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:553)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:254)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:32)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:528)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:149)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:217)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:143)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1466)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1529)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2239)
Caused by: oracle.net.ns.NetException: The Network Adapter could not establish the connection
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:439)
	at oracle.net.resolver.AddrResolution.resolveAndExecute(AddrResolution.java:454)
	at oracle.net.ns.NSProtocol.establishConnection(NSProtocol.java:693)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:251)
	at oracle.jdbc.driver.T4CConnection.connect(T4CConnection.java:1140)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:340)
	... 10 more
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at oracle.net.nt.TcpNTAdapter.connect(TcpNTAdapter.java:149)
	at oracle.net.nt.ConnOption.connect(ConnOption.java:133)
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:405)
	... 15 more
[2017-10-12 11:08:57,603][TxnService-4]com.ods.db.DbPool[62][DEBUG]: 数据库连接获取完毕
[2017-10-12 11:09:06,632][TxnService-1]com.ods.db.DbPool[60][ERROR]: 获取数据库连接失败
com.alibaba.druid.pool.GetConnectionTimeoutException: wait millis 60001, active 0, maxActive 10, creating 0
	at com.alibaba.druid.pool.DruidDataSource.getConnectionInternal(DruidDataSource.java:1377)
	at com.alibaba.druid.pool.DruidDataSource.getConnectionDirect(DruidDataSource.java:1198)
	at com.alibaba.druid.filter.FilterChainImpl.dataSource_connect(FilterChainImpl.java:4619)
	at com.alibaba.druid.filter.stat.StatFilter.dataSource_getConnection(StatFilter.java:666)
	at com.alibaba.druid.filter.FilterChainImpl.dataSource_connect(FilterChainImpl.java:4615)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1176)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1168)
	at com.ods.db.DbPool.getConnection(DbPool.java:57)
	at com.ods.db.DbQuery.excuteQuery(DbQuery.java:36)
	at com.ods.transaction.test.transaction(test.java:24)
	at com.ods.service.TxnService.run(TxnService.java:175)
Caused by: java.sql.SQLRecoverableException: IO 错误: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:489)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:553)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:254)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:32)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:528)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:149)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:217)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:143)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1466)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1529)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2239)
Caused by: oracle.net.ns.NetException: The Network Adapter could not establish the connection
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:439)
	at oracle.net.resolver.AddrResolution.resolveAndExecute(AddrResolution.java:454)
	at oracle.net.ns.NSProtocol.establishConnection(NSProtocol.java:693)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:251)
	at oracle.jdbc.driver.T4CConnection.connect(T4CConnection.java:1140)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:340)
	... 10 more
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at oracle.net.nt.TcpNTAdapter.connect(TcpNTAdapter.java:149)
	at oracle.net.nt.ConnOption.connect(ConnOption.java:133)
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:405)
	... 15 more
[2017-10-12 11:09:06,634][TxnService-1]com.ods.db.DbPool[62][DEBUG]: 数据库连接获取完毕
[2017-10-12 11:09:53,139][TxnService-3]com.ods.db.DbPool[60][ERROR]: 获取数据库连接失败
com.alibaba.druid.pool.GetConnectionTimeoutException: wait millis 60001, active 0, maxActive 10, creating 0
	at com.alibaba.druid.pool.DruidDataSource.getConnectionInternal(DruidDataSource.java:1377)
	at com.alibaba.druid.pool.DruidDataSource.getConnectionDirect(DruidDataSource.java:1198)
	at com.alibaba.druid.filter.FilterChainImpl.dataSource_connect(FilterChainImpl.java:4619)
	at com.alibaba.druid.filter.stat.StatFilter.dataSource_getConnection(StatFilter.java:666)
	at com.alibaba.druid.filter.FilterChainImpl.dataSource_connect(FilterChainImpl.java:4615)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1176)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1168)
	at com.ods.db.DbPool.getConnection(DbPool.java:57)
	at com.ods.db.DbQuery.excuteQuery(DbQuery.java:36)
	at com.ods.transaction.test.transaction(test.java:24)
	at com.ods.service.TxnService.run(TxnService.java:175)
Caused by: java.sql.SQLRecoverableException: IO 错误: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:489)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:553)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:254)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:32)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:528)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:149)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:217)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:143)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1466)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1529)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2239)
Caused by: oracle.net.ns.NetException: The Network Adapter could not establish the connection
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:439)
	at oracle.net.resolver.AddrResolution.resolveAndExecute(AddrResolution.java:454)
	at oracle.net.ns.NSProtocol.establishConnection(NSProtocol.java:693)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:251)
	at oracle.jdbc.driver.T4CConnection.connect(T4CConnection.java:1140)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:340)
	... 10 more
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at oracle.net.nt.TcpNTAdapter.connect(TcpNTAdapter.java:149)
	at oracle.net.nt.ConnOption.connect(ConnOption.java:133)
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:405)
	... 15 more
[2017-10-12 11:09:53,140][TxnService-3]com.ods.db.DbPool[62][DEBUG]: 数据库连接获取完毕
[2017-10-12 11:10:21,894][TxnService-2]com.ods.db.DbPool[60][ERROR]: 获取数据库连接失败
com.alibaba.druid.pool.GetConnectionTimeoutException: wait millis 60000, active 0, maxActive 10, creating 0
	at com.alibaba.druid.pool.DruidDataSource.getConnectionInternal(DruidDataSource.java:1377)
	at com.alibaba.druid.pool.DruidDataSource.getConnectionDirect(DruidDataSource.java:1198)
	at com.alibaba.druid.filter.FilterChainImpl.dataSource_connect(FilterChainImpl.java:4619)
	at com.alibaba.druid.filter.stat.StatFilter.dataSource_getConnection(StatFilter.java:666)
	at com.alibaba.druid.filter.FilterChainImpl.dataSource_connect(FilterChainImpl.java:4615)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1176)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1168)
	at com.ods.db.DbPool.getConnection(DbPool.java:57)
	at com.ods.db.DbQuery.excuteQuery(DbQuery.java:36)
	at com.ods.transaction.test.transaction(test.java:24)
	at com.ods.service.TxnService.run(TxnService.java:175)
Caused by: java.sql.SQLRecoverableException: IO 错误: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:489)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:553)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:254)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:32)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:528)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:149)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:217)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:143)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1466)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1529)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2239)
Caused by: oracle.net.ns.NetException: The Network Adapter could not establish the connection
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:439)
	at oracle.net.resolver.AddrResolution.resolveAndExecute(AddrResolution.java:454)
	at oracle.net.ns.NSProtocol.establishConnection(NSProtocol.java:693)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:251)
	at oracle.jdbc.driver.T4CConnection.connect(T4CConnection.java:1140)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:340)
	... 10 more
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at oracle.net.nt.TcpNTAdapter.connect(TcpNTAdapter.java:149)
	at oracle.net.nt.ConnOption.connect(ConnOption.java:133)
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:405)
	... 15 more
[2017-10-12 11:10:21,895][TxnService-2]com.ods.db.DbPool[62][DEBUG]: 数据库连接获取完毕
[2017-10-12 11:10:47,503][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:10:47,511][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:11:17,433][TxnService-0]com.ods.db.DbPool[60][ERROR]: 获取数据库连接失败
java.sql.SQLRecoverableException: IO 错误: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:489)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:553)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:254)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:32)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:528)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:149)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:217)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:143)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1466)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1529)
	at com.alibaba.druid.pool.DruidDataSource.init(DruidDataSource.java:822)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1172)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1168)
	at com.ods.db.DbPool.getConnection(DbPool.java:57)
	at com.ods.db.DbQuery.excuteQuery(DbQuery.java:36)
	at com.ods.transaction.test.transaction(test.java:24)
	at com.ods.service.TxnService.run(TxnService.java:175)
Caused by: oracle.net.ns.NetException: The Network Adapter could not establish the connection
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:439)
	at oracle.net.resolver.AddrResolution.resolveAndExecute(AddrResolution.java:454)
	at oracle.net.ns.NSProtocol.establishConnection(NSProtocol.java:693)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:251)
	at oracle.jdbc.driver.T4CConnection.connect(T4CConnection.java:1140)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:340)
	... 16 more
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at oracle.net.nt.TcpNTAdapter.connect(TcpNTAdapter.java:149)
	at oracle.net.nt.ConnOption.connect(ConnOption.java:133)
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:405)
	... 21 more
[2017-10-12 11:11:17,434][TxnService-0]com.ods.db.DbPool[62][DEBUG]: 数据库连接获取完毕
[2017-10-12 11:11:49,498][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:11:49,507][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:25:47,103][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:25:47,110][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:25:56,380][TxnService-2]com.ods.db.DbPool[62][DEBUG]: 数据库连接获取完毕
[2017-10-12 11:28:45,562][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:28:45,570][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:29:12,312][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:29:12,320][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:32:36,100][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:32:36,109][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:32:40,728][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:32:40,736][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:32:42,954][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:32:42,973][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:32:43,003][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:32:43,033][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:32:43,812][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:32:43,914][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:32:44,051][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:32:44,070][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:32:44,083][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:32:44,126][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:35:51,886][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:35:51,894][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:36:01,996][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:36:02,004][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:36:34,486][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:36:34,494][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:36:40,232][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:36:40,241][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:36:50,279][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:36:50,286][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:38:28,688][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:38:32,609][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:38:41,029][TxnService-4]com.ods.db.DbPool[62][DEBUG]: 数据库连接获取完毕
[2017-10-12 11:42:59,731][TxnService-4]com.ods.db.DbPool[60][ERROR]: 获取数据库连接失败
com.alibaba.druid.pool.GetConnectionTimeoutException: wait millis 60000, active 0, maxActive 10, creating 0
	at com.alibaba.druid.pool.DruidDataSource.getConnectionInternal(DruidDataSource.java:1377)
	at com.alibaba.druid.pool.DruidDataSource.getConnectionDirect(DruidDataSource.java:1198)
	at com.alibaba.druid.filter.FilterChainImpl.dataSource_connect(FilterChainImpl.java:4619)
	at com.alibaba.druid.filter.stat.StatFilter.dataSource_getConnection(StatFilter.java:666)
	at com.alibaba.druid.filter.FilterChainImpl.dataSource_connect(FilterChainImpl.java:4615)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1176)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1168)
	at com.ods.db.DbPool.getConnection(DbPool.java:57)
	at com.ods.db.DbQuery.excuteQuery(DbQuery.java:36)
	at com.ods.transaction.test.transaction(test.java:24)
	at com.ods.service.TxnService.run(TxnService.java:175)
Caused by: java.sql.SQLRecoverableException: IO 错误: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:489)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:553)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:254)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:32)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:528)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:149)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:217)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:143)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1466)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1529)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2239)
Caused by: oracle.net.ns.NetException: The Network Adapter could not establish the connection
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:439)
	at oracle.net.resolver.AddrResolution.resolveAndExecute(AddrResolution.java:454)
	at oracle.net.ns.NSProtocol.establishConnection(NSProtocol.java:693)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:251)
	at oracle.jdbc.driver.T4CConnection.connect(T4CConnection.java:1140)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:340)
	... 10 more
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at oracle.net.nt.TcpNTAdapter.connect(TcpNTAdapter.java:149)
	at oracle.net.nt.ConnOption.connect(ConnOption.java:133)
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:405)
	... 15 more
[2017-10-12 11:42:59,733][TxnService-4]com.ods.db.DbPool[62][DEBUG]: 数据库连接获取完毕
[2017-10-12 11:43:21,487][TxnService-1]com.ods.db.DbPool[60][ERROR]: 获取数据库连接失败
com.alibaba.druid.pool.GetConnectionTimeoutException: wait millis 60000, active 0, maxActive 10, creating 0
	at com.alibaba.druid.pool.DruidDataSource.getConnectionInternal(DruidDataSource.java:1377)
	at com.alibaba.druid.pool.DruidDataSource.getConnectionDirect(DruidDataSource.java:1198)
	at com.alibaba.druid.filter.FilterChainImpl.dataSource_connect(FilterChainImpl.java:4619)
	at com.alibaba.druid.filter.stat.StatFilter.dataSource_getConnection(StatFilter.java:666)
	at com.alibaba.druid.filter.FilterChainImpl.dataSource_connect(FilterChainImpl.java:4615)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1176)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1168)
	at com.ods.db.DbPool.getConnection(DbPool.java:57)
	at com.ods.db.DbQuery.excuteQuery(DbQuery.java:36)
	at com.ods.transaction.test.transaction(test.java:24)
	at com.ods.service.TxnService.run(TxnService.java:175)
Caused by: java.sql.SQLException: Listener refused the connection with the following error:
ORA-12505, TNS:listener does not currently know of SID given in connect descriptor
 
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:489)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:553)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:254)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:32)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:528)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:149)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:217)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:143)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1466)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1529)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2239)
Caused by: oracle.net.ns.NetException: Listener refused the connection with the following error:
ORA-12505, TNS:listener does not currently know of SID given in connect descriptor
 
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:399)
	at oracle.jdbc.driver.T4CConnection.connect(T4CConnection.java:1140)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:340)
	... 10 more
[2017-10-12 11:43:21,487][TxnService-1]com.ods.db.DbPool[62][DEBUG]: 数据库连接获取完毕
[2017-10-12 11:45:44,338][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:45:51,196][TxnService-3]com.ods.db.DbPool[62][DEBUG]: 数据库连接获取完毕
[2017-10-12 11:46:20,052][TxnService-3]com.ods.db.DbPool[62][DEBUG]: 数据库连接获取完毕
[2017-10-12 11:49:00,083][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:49:07,616][TxnService-2]com.ods.db.DbPool[62][DEBUG]: 数据库连接获取完毕
[2017-10-12 11:50:31,745][TxnService-2]com.ods.db.DbPool[62][DEBUG]: 数据库连接获取完毕
[2017-10-12 11:53:09,631][TxnService-4]com.ods.db.DbPool[62][DEBUG]: 数据库连接获取完毕
[2017-10-12 11:55:34,568][TxnService-4]com.ods.db.DbPool[62][DEBUG]: 数据库连接获取完毕
[2017-10-12 11:56:27,295][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 11:56:30,745][TxnService-0]com.ods.db.DbPool[62][DEBUG]: 数据库连接获取完毕
[2017-10-12 12:00:10,555][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 12:00:15,817][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 12:00:20,993][TxnService-0]com.ods.db.DbPool[62][DEBUG]: 数据库连接获取完毕
[2017-10-12 14:58:49,247][TxnService-2]com.ods.db.DbPool[62][DEBUG]: 数据库连接获取完毕
[2017-10-12 15:04:34,814][TxnService-0]com.ods.db.DbPool[62][DEBUG]: 数据库连接获取完毕
[2017-10-12 15:05:29,775][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 15:06:06,348][TxnService-2]com.ods.db.DbPool[62][DEBUG]: 数据库连接获取完毕
[2017-10-12 15:12:07,505][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 15:12:39,953][main]com.ods.db.DbPool[72][INFO ]: 数据库连接池建立完毕
[2017-10-12 15:12:57,389][TxnService-4]com.ods.db.DbPool[62][DEBUG]: 数据库连接获取完毕
